### DISCUSSION TITLE
Ctl+Alt+P not finding source members with period in the name

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1273] [IBM i]
Within our naming convention we have certain scenarios where a period is a part of the source member name; some start with p. , some start with f. , and some start with j. for example.  When these source members are found and opened via a single left click within a filter, they open fine.  However, when I use Ctl+Alt+P and type out the name I get this error:
![image](https://user-images.githubusercontent.com/132368707/236446651-ee4047cb-a43e-42d8-8732-1fb761257d93.png)

I've verified that I'm typing the name of the source member correctly.  I cannot imagine what I'm doing wrong, but if the answer is that I am doing something wrong, then I'm all ears.  But I guess we need a tweak to handle certain non-alphabetic characters in source names.

Thanks in advance,
Gregory

**Comment by worksofliam** [comment]
When you enter the path to the member, are you also giving an extension? Give that a try.

**Reply by Gregory1701** [reply]
Not sure what changed, but now those source members are coming up without typing the extension.  There are a lot of source members in this particular library/source physical file and it does get laggy when I type in the first two characters of the source member name.  But that wasn't happening before...  Ah well, sorry to bother you.  Nothing to see here... 'these aren't the drones you're looking for', 'you may proceed'...
